app/templates/client/dev/components/todo/todo.js

```diff
@@ -1,51 +1,75 @@
-;(function(angular)
-{
-  "use strict";
-
-  angular
-    .module('myAwesomeApp')
-    .controller('TodoController', ['$log', 'Todo', 'TodoDAO', function($log, Todo, TodoDAO)
-    {
-      var self = this;
-
-      self.todo = new Todo();
-      self.todos = [];
-
-      self.createTodo = function(todo)
-      {
-        var _onSuccess = function(newTodo)
-        {
-          self.todos.push(newTodo);
-          self.todo = new Todo();
-        };
+(function () {
```

Comment:
On this point, I think we should. They're basically the same, but the former is more trustable- specially when you're concating your code with other people's. I just ran to this problem when using ngMessages 1.4.x, by the way.